src/modules/login/login.js

```diff
@@ -74,14 +74,8 @@
         function validateUiFields() {
```

Comment:
@gnehapk I think we can remove this function as well.

Reply:
@a2batic We are validating username and password here in this function.